modules/datahandling/filerequests_test.go

```diff
@@ -258,8 +251,9 @@ func TestFileChangeRequest_Process(t *testing.T) {
 	req.Resource = "File"
 	req.Method = "Change"
 	req.FileID = fileid
-	req.Changes = []string{"change 1"}
-	req.BaseFileVersion = newFileVersion
+	req.Changes = []string{"v0:\n0:+1:a"}
+
+	baseFileVersion := int64(1)
 
```

Comment:
yeah @ObsessiveOrange has the right idea, the part above was building the request, whereas resetting the function call count is it's own independent operation